src/CloudFoundry.VisualStudio/CloudFoundry.VisualStudioPackage.cs

```diff
@@ -68,10 +68,15 @@ public sealed class CloudFoundry_VisualStudioPackage : Package
         /// </summary>
         public CloudFoundry_VisualStudioPackage()
         {
+            errorList = new ErrorListProvider(this);
             this.dte = (DTE)CloudFoundry_VisualStudioPackage.GetGlobalService(typeof(DTE));
             Debug.WriteLine(string.Format(CultureInfo.InvariantCulture, "Entering constructor for: {0}", this.ToString()));
         }
 
+        public static ErrorListProvider GetErrorList()
```

Comment:
Please rename to `GetErrorListWindow` or `GetErrorListView` or something similar. Right now, its meaning is too close to "get me a list of errors".